index.js

```diff
@@ -0,0 +1,144 @@
+"use strict";
+
+const startPage = document.querySelector(".page-start"),
+    startBtn = startPage.querySelector(".startBtn-start");
+
+const playingPage = document.querySelector(".page-playing"),
+    digitForm = playingPage.querySelector(".form-playing"),
+    digitInput = playingPage.querySelector(".input-playing"),
+    scoreBoard = playingPage.querySelector(".scoreboard-playing");
+
+const endPage = document.querySelector(".page-end"),
+    leaderBoard = endPage.querySelector(".leaderBoard-end"),
+    restartBtn = endPage.querySelector(".restartBtn-end");
+
+const VAILD_DIGIT_LENGTH = 3;
+const HIDING = "hiding";
+
+let wholePage = [startPage, playingPage, endPage];
+
+let ballCount,
+    strikeCount,
+    inningCount;
+
+let invaildMatch = 0; //유효하지 않은 숫자 입력시 inningCount가 올라가는 것을 방지 1은 유효하지않은게임 0은 유효한 게임
+
+let r3d = []; // 랜덤 숫자 3개
+let i3d = []; // 입력받은 숫자 3개
+
+function set3RandomDigit() {
+    for (let i = 0; i < 3; i++) {
+        r3d[i] = parseInt(Math.random() * 9);
+    }
+    if (r3d[0] === r3d[1] || r3d[0] === r3d[2] || r3d[1] === r3d[2]) {
```

Comment:
숫자가 중복되는 경우, 다시 랜덤 숫자를 받는 로직을 잘 구현해주신 것 같습니다.👍
다만, 숫자가 세자리에서 네자리만 되어도 경우의 수가 많아져서 for문 등을 통해 비교 로직을 작성해봐도 좋을 것 같아요.
그리고 만약 조건문이 길다면 아래와 같이 표현해도 좋습니다.
```suggestion
    const hasSameNumber = r3d[0] === r3d[1] || r3d[0] === r3d[2] || r3d[1] === r3d[2];
    if (hasSameNumber) {
```